rocAL/rocAL/include/meta_data/meta_node_resize_mirror_normalize.h

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright (c) 2019 - 2023 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
If this file in newly introduced change copyright to 2023

Reply:
Change copyright to 2023